pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -2233,9 +2245,14 @@ ParticipantStatusTrackingInfo.Kit:
         - RECEIVED
         - UNKNOWN
     sentAt:
-      type: integer
+      type: string
+      description: ISO-8601 date
     receivedBackAt:
-      type: integer
+      type: string
+      description: ISO-8601 date
```

Comment:
Again, what's the technical reason for breaking the API here?

Reply:
@YuriyPavlovBI Let's use the same approach here as in the [previous comment](https://github.com/broadinstitute/ddp-study-server/pull/723/files/e0f3762bed10f10ac8ae7274f9f5573bee528fa5#r622223630). Leave `sentAt` and `receivedBackAt` as `integer` type fields, mark them as deprecated in OpenAPI, and add the `sentAtDate` and `receivedBackAtDate` as `string` type fields, with the `format` `ISO-8601 Date`

Reply:
Rolled back